masschallenge/django-accelerator
# [AC-6414] Running make migrations creates a sitetree migration that changes the tree foreign key in NavTreeItem

### Changes introduced in [AC-6414](https://masschallenge.atlassian.net/browse/AC-6414):
- Ensure the tree field in the NavTreeItem references our custom Tree model i.e NavTree model

### How to test
- On the development branch, if you run `make migrations` a migration is created resulting in the tree field in NavTreeItem referencing the default Tree model `sitetree.Tree` rather than our custom model `mc.NavTree`
- On this branch if you run `make migrations` a migration should associate the tree field in NavTreeItem to our custom model `mc.NavTree`
- You should still be able to run add NavTreeItems to NavTrees in admin `http://localhost:8181/admin/mc/navtree/`
